packages/ui/components/Pages/Program/EditProgramModal.tsx

```diff
@@ -18,16 +33,14 @@ const EditProgramModal = ({
           setEditProgramModalOpen(false);
         }}
       />
-      <form className="flex flex-col px-2 mt-8 pb-6">
-        <InputField
-          label="Program Name"
-          value="SSMD Gene Therapy"
-          className="text-sm text-gray-500 text-light mt-8"
-          name="Program Name"
-          placeholder=""
-          reference={inputRef}
-        />
-
+      <Form
+        {...layout}
+        name="basic"
+        initialValues={{ remember: true }}
+        onFinish={Demo}
+        onFinishFailed={onFinishFailed}
+      >
+        <Input placeholder="Basic usage" />
```

Comment:
Please wrap all inputs in Form.Item as it makes things more accessible among other benefits